.travis.yml

```diff
@@ -63,6 +77,7 @@ jobs:
 
 env:
   global:
+    - TRAVIS_JDK=8.0.202.hs-adpt
```

Comment:
I'm not sure I'm a fan of baking this exact version number into here. Typically, we just ask for 8/9/12 and let external config in the CI platform pick the latest minor release for that.

Reply:
Travis's native support for JDK installation comes from:

https://github.com/sormuras/bach/blob/master/install-jdk.sh

Should we contribute AdoptOpenJDK support to that script?

Reply:
We can do

```
sdk list java | grep -o "8\..*hs-adpt" | tail -1
```

to let sdkman decide the patch version.

Reply:
If the bach maintainer is willing to maintain AdoptOpenJDK links, that would be great.
See also
- https://github.com/travis-ci/travis-cookbooks/issues/976
- https://github.com/travis-ci/travis-ci/issues/9368
- https://github.com/travis-ci/travis-build/pull/1347

Reply:
Looks like `install_jdk.sh` already supports Adopt OpenJDK: 

Example: https://github.com/sormuras/sormuras.github.io/blob/master/.travis.yml#L27

The only catch is that it doesn't have an alias for it that can be specified in the `jdk: openjdk8` directive in `.travis.yml`.

This URL resolves to the latest release of AdoptOpenJDK 8: https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl=hotspot&os=linux&arch=x64&release=latest&heap_size=normal&type=jdk

Reply:
Added patch detection here - b45f9aa61d553b94e10fa297017c22455d3f9fe0